start-from-zero/index.css

```diff
@@ -246,21 +350,208 @@ textarea {
 
 #select {
     float: right;
+    -webkit-appearance: none; 
+    -moz-appearance: none;
+    appearance: none;   
+    background-image: url(assets/down-arrow.png);
+    background-repeat: no-repeat;
+    background-position: 90% 50%;
+    background-size: 13px;
+    position: relative;
+    height: 20px;
+    
+}
+
+
+
+textarea {
+    width: 99.7%;
```

Comment:
I totally forgot about this. It was just a quick fix. I still need to figure out why it doesn't align on the right side like all the other elements.